runtime/lua/vim/shared.lua

```diff
@@ -347,6 +347,22 @@ function vim.tbl_add_reverse_lookup(o)
   return o
 end
 
+--- Returns a nested value if it exists, else false.
+---
+---@param t Table to index
+---@param keys table of nested attributes
+---
+---@returns nested value indexed by key if it exists, else `false`.
+function vim.tbl_get(o, keys)
```

Comment:
This is not correct, need to check k is empty and return false if so, not sure about this one being a general function.